src/index.js

```diff
@@ -103,6 +102,28 @@ const loadMargaritaInstruction = (userQuery) => {
 
             }
           }          
+          document.querySelector(".cbox1").addEventListener('change', (event) => {
+            if (event.target.checked) {
+              document.getElementById("infoBebidaDiv").style.border = "4px solid greenyellow";
+            } else {
+              document.getElementById("infoBebidaDiv").style.border = "none";
+            }
+          })
+          document.querySelector(".cbox1").addEventListener('change', (event) => {
```

Comment:
You are adding two change events to the same element, you can use only one event to do both things,

Reply:
ok, I don't remember how to do that but I will check my notes